### PR TITLE
Update table

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | f38d905 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-543.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/connect_manage_environ/_index.md [hunk 1]
```diff
@@ -94,6 +109,7 @@ For detailed information about agent configuration, features and limitations, se
 * [Discovery Agent for GitLab Repository](/docs/connect_manage_environ/connect_gitlab_repository/)
 * [Discovery Agent for Kafka Cluster](/docs/connect_manage_environ/connect_kafka_cluster/)
 * [Discovery Agent for SwaggerHub](/docs/connect_manage_environ/connect_swaggerhub/)
+* [Discovery Agent for Backstageb](/docs/connect_manage_environ/connect_backstage/)
```
Comment (jcollins-axway) [inline content/en/docs/connect_manage_environ/_index.md:112]: Backstageb*

**Review by jcollins-axway — APPROVED** [review]

File: content/en/docs/connect_manage_environ/_index.md (hunk 1 above)
Comment (robertengaxway) [reply content/en/docs/connect_manage_environ/_index.md]: fixed

**Review by robertengaxway — APPROVED** [review]
approved

**Pull request merged**